packages/components/src/FilterBar/context/reducer/filterBarStateReducer.ts

```diff
@@ -35,5 +39,12 @@ export const filterBarStateReducer = <ValuesMap extends FiltersValues>(
     case "deactivate_filter":
       state.activeFilterIds.delete(action.id)
       return { ...state }
+
+    case "clear_all_filters":
+      state.activeFilterIds.forEach(id => {
+        if (state.filters[id].isRemovable) state.activeFilterIds.delete(id)
```

Comment:
Should this logic not also be part of the deactivate action? You shouldn't be able to deactivate non removable filters right?

And then, are you able to then dispatch the deactivate_filter action here rather than repeating? I'm imagining that this action would be better by dispatching a combo of other actions.